app/views/news_tematica/news_tematicas/elegir_contenidos.html.haml

```diff
@@ -36,10 +41,13 @@
   &nbsp; &nbsp;
   = submit_tag('Generar HTML', class: 'boton_reborde_naranja_grande')
 
-:javascript
-  jQuery(function($) {
-    $('.js_ver_resto_titulares').click(function() {
-      $('.js_resto_titulares').slideDown();
-      $(this).hide();
-    });
-  });
+:coffee
```

Comment:
Disclaimer: I wasn't asking @d2bit  to change this, but asking @all for a consensus about good practices...